### PR TITLE
docs(video): consistent library title

### PR DESCRIPTION
**Pull request by coryan**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/11773)

### CODE REVIEW

File: google/cloud/video/CMakeLists.txt [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 include(GoogleapisConfig)
 set(DOXYGEN_PROJECT_NAME "Transcoder API C++ Client")
-set(DOXYGEN_PROJECT_BRIEF "A C++ Client Library for the Transcoder API")
+set(DOXYGEN_PROJECT_BRIEF "A C++ Client Library for the several Video Services")
```
Comment (devbww) [inline google/cloud/video/CMakeLists.txt:19]: s/for the several/for several/

File: google/cloud/video/quickstart/WORKSPACE.bazel [hunk 2]
```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# A minimal WORKSPACE file showing how to use the Transcoder API
-# C++ client library in Bazel-based projects.
+# A minimal WORKSPACE file showing how to use the video services  C++ client
```
Comment (devbww) [inline google/cloud/video/quickstart/WORKSPACE.bazel:15]: `s/<sp><sp>/<sp>/`

**Review by devbww — APPROVED** [review]

File: google/cloud/video/CMakeLists.txt (hunk 1 above)
Comment (coryan) [reply google/cloud/video/CMakeLists.txt]: Fixed

File: google/cloud/video/quickstart/WORKSPACE.bazel (hunk 2 above)
Comment (coryan) [reply google/cloud/video/quickstart/WORKSPACE.bazel]: Fixed

**Pull request merged**